toolset/benchmark/benchmarker.py

```diff
@@ -650,6 +650,19 @@ def exit_with_code(code):
         time.sleep(5)
 
         ##########################################################
+        # Remove contents of  /tmp folder
+        ##########################################################
+        try:
+          filelist = [ f for f in os.listdir("/tmp") ]
+	  for f in filelist:
```

Comment:
I just wanted to let you know that there are a couple of tab characters mixed in here. There's one at the beginning of the line here and two lines down (`os.remove("/tmp/ + f")`).